source/slang/slang-check-constraint.cpp

```
@@ -499,6 +499,135 @@ bool addHasDiffTypeInfoWitnessToArgs(
+    ShortList<QualType, 8> types;
+};
+
+bool constraintSystemSolverUnpackArgs(
+    SemanticsVisitor* visitor,
+    List<GenericDecl*>& genericDecls,
+    ArrayView<Val*> knownGenericArgs,
+    Dictionary<Decl*, ShortList<SolvedArg>>& solvedArgs,
+    Dictionary<Decl*, ShortList<Val*>>& args)
+{
+    ASTBuilder* astBuilder = visitor->getASTBuilder();
+
```

Cppcheck (2.20.0)

[information] Include file

(missingIncludeSystem)